bin/z-gerrit

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env bash
+
+set -e
+
+if [ -n "$Z_LIB" ]; then
+    . $Z_LIB/env.sh
+else
+    echo "ERROR Incorrectly configured environment, Z_LIB not set" >&2
+    exit 1
+fi
+
+. $Z_LIB/gerrit.functions
+
+usage="Usage: $(basename $0) [-d] [gerrit_command]
```

Comment:
This variable isn't declared with appropriate flags (e.g. read-only). I actually prefer putting usage in a function that echos to stderr or stdout depending on when and how it is called (taking a parameter for the stream to output to).